sdks/wasm/src/driver.c

```diff
@@ -488,6 +489,28 @@ class_is_task (MonoClass *klass)
 	return 0;
 }
 
+MonoClass* mono_get_uri_class(MonoException** exc) 
+{
+	MonoAssembly* assembly = mono_wasm_assembly_load ("WebAssembly.Bindings");
+	if (!assembly)
+		return NULL;
+	MonoClass* klass = mono_wasm_assembly_find_class(assembly, "WebAssembly", "Runtime");
```

Comment:
If we're already going to the trouble here of looking up a class in an assembly and calling a method, why don't we just look System.Uri up directly?

Reply:
I tried looking it up with `mono_class_from_name (mono_get_corlib(), "System", "Uri");` but no class is found.

Reply:
That might be because it doesn't always live in mscorlib.dll, it looks like according to the docs site it can live in System or System.Runtime instead.

Reply:
Addressed in recent commits